### ISSUE TITLE
TypeError: _set_ticklabels() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by iranmdl** [AlexandrovLab/SigProfilerExtractor#105]
Hi! I am trying to run sigprofilerextractor using the following  command: 
```
data = "/home/pipelines/SigProfilerMatrixGenerator_output/input"
sig.sigProfilerExtractor("vcf", "sigProfilerExtractor_output", data, reference_genome="GRCh38", exome=True, minimum_signatures=1, maximum_signatures=10)
```

However, I am getting the following error:
```
Time taken to collect 500 iterations for 10 signatures is 434.53 seconds
Optimization time is 5.282233238220215 seconds
The reconstruction error is 0.0789, average process stability is 0.16 and
the minimum process stability is -0.22 for 10 signatures


Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/miniconda3/lib/python3.8/site-packages/SigProfilerExtractor/sigpro.py", line 764, in sigProfilerExtractor
    solution, all_stats = sub.stabVsRError(layer_directory+"/All_solutions_stat.csv", layer_directory, title, all_similirities_list, mtype=mutation_type, stability=stability, min_stability=min_stability, combined_stability=combined_stability)
  File "/home/miniconda3/lib/python3.8/site-packages/SigProfilerExtractor/subroutines.py", line 1840, in stabVsRError
    ax1.set_xticklabels(np.arange(min(t), max(t)+1, 1),list(), rotation=30)
  File "/home/miniconda3/lib/python3.8/site-packages/matplotlib/axes/_base.py", line 75, in wrapper
    return get_method(self)(*args, **kwargs)
TypeError: _set_ticklabels() takes 2 positional arguments but 3 were given
```
Logs:
[JOB_METADATA.txt](https://github.com/AlexandrovLab/SigProfilerExtractor/files/7581747/JOB_METADATA.txt)

Any help is very appreciated :)

**Comment by mdbarnesUCSD** [comment]
We are working on updating the code to use the most recent release of matplotlib. Until then, please use matplotlib v3.4.3:
`pip install matplotlib==3.4.3`

**Comment by iranmdl** [comment]
Thank you! Issue fixed using matplotlib v3.4.3 :)
